MovieAppPetProject/MovieAppPetProject/HomeScreen/PosterCellCollectionViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PosterCellCollectionViewCell.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/04/11.
+//
+
+import UIKit
+
+class PosterCellCollectionViewCell: UICollectionViewCell {
+    
+    
```

Comment:
all this extra space?
Please don't just ignore the swiftlint warnings.
It should be picking it up. Either it is not set up correctly or you are ignoring the warnings.
Please have 0 swiftlint warnings when going from develop to main

Reply:
cleaned up 👍